drivers/soundwire/intel_bus_common.c

```diff
@@ -77,87 +77,64 @@ int intel_start_bus_after_reset(struct sdw_intel *sdw)
 	struct device *dev = sdw->cdns.dev;
 	struct sdw_cdns *cdns = &sdw->cdns;
 	struct sdw_bus *bus = &cdns->bus;
-	bool clock_stop0;
 	int status;
 	int ret;
 
 	/*
-	 * An exception condition occurs for the CLK_STOP_BUS_RESET
-	 * case if one or more masters remain active. In this condition,
-	 * all the masters are powered on for they are in the same power
-	 * domain. Master can preserve its context for clock stop0, so
-	 * there is no need to clear slave status and reset bus.
+	 * make sure all Slaves are tagged as UNATTACHED and
+	 * provide reason for reinitialization
```

Comment:
Not sure if it is the right direction. The clock may remain active, and the peripherals may remain attached when bus stops. It seems not be necessary to reset in that case.